sdk/ml/azure-ai-ml/tests/test_configs/flows/flow_with_meta/compare.md

```diff
@@ -0,0 +1,29 @@
+| field name          | can be ignore? | flow.dag.yaml                                 | flow.meta.yaml                                 | run.yaml                     | standalone file       |
+| ------------------- | -------------- | --------------------------------------------- | ---------------------------------------------- | ---------------------------- | --------------------- |
```

Comment:
may also add the flow.dag.yaml only fields to this table

Reply:
removed this file and reformat the solution